features/conversation/conversation-message/conversation-message-context-menu/conversation-message-context-menu-container.tsx

```diff
@@ -85,16 +86,22 @@ export const MessageContextMenuContainer = memo(
       theme,
       verticalSpaceBetweenSections,
       translateYAV,
+      windowHeight,
     ]);
 
     const animatedStyle = useAnimatedStyle(() => ({
       transform: [{ translateY: translateYAV.value }],
     }));
 
+    // On Android, we need to account for the status bar height by moving the menu down
+    const statusBarHeight =
+      Platform.OS === "android" ? StatusBar.currentHeight || 0 : 0;
+
     const distanceFromTop =
       itemRectY -
       MESSAGE_CONTEXT_MENU_ABOVE_MESSAGE_REACTIONS_HEIGHT -
-      verticalSpaceBetweenSections;
+      verticalSpaceBetweenSections +
+      (Platform.OS === "android" ? statusBarHeight : 0);
```

Comment:
Shouldn't we use `safeAreaInsets` top  like we're doing at other places? I know if we do that it might fuck iOS but can we undersatnd why? I feel using StatusBar height isn't good here since StatusBar height isn't really stable from what I know.

Reply:
Also I thought the problem was that only the bottom part of the context view menu was off. Not the entire context menu. Might be wrong here. Can't reproduce on my device since the app crashes.